operations/package_repository_management/Troubleshoot_Nexus.md

```diff
@@ -5,6 +5,7 @@ This page contains general Nexus troubleshooting topics.
 - [`lookup registry.local: no such host`](#lookup-registrylocal-no-such-host)
 - [`Error initiating layer upload` ... `in registry.local`](#error-initiating-layer-upload--in-registrylocal)
 - [`error: not ready: https://packages.local`](#error-not-ready-httpspackageslocal)
+- [`Unable to login with Keycloak accounts`](#unable-to-login-with-keycloak-accounts)
```

Comment:
Only need to enclose it in backticks if we are quoting the exact error message being seen by the user, which I don't think is the case here.

```suggestion
- [Unable to login with Keycloak accounts](#unable-to-login-with-keycloak-accounts)
```